cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceContextConfig.java

```diff
@@ -126,6 +130,15 @@ public String getWorkflowBatch() {
     return jobContext.getConfiguration().get(HCONF_ATTR_WORKFLOW_BATCH);
   }
 
+  private void setAdapterName(@Nullable String adapterName) {
```

Comment:
I could go either way on this. This same class uses two different styles of setting nulls. ``setWorkflowBatch`` allows callers to pass null and handles it internally, while ``setInputSelection`` expects callers to handle nulls before calling.

Following your suggestion, but applying the same to ``setWorkflowBatch`` as well.